llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -724,7 +901,21 @@ def forward(
                 attention_mask,
             )
 
+        layer_kv_cache_dict = {}
         for b_idx, block in enumerate(self.blocks):
+            attn_block = block.attn if hasattr(
```

Comment:
this should check the config.fuse_norm_attn_norm key instead of checking for the attr

Reply:
Done.